login-fire-common-behavior.html

```diff
@@ -260,7 +262,12 @@
       _user: {
         type: Object,
         observer: '_userChanged'
-      }
+      },
+
+      _showOneButton: {
```

Comment:
This is used in both login-fire and login-fire-social. When the user can sign out, instead of showing a bunch of buttons that allow sign out, it removes all the buttons and just shows 1 that can sign out. I added comments to explain it. Thanks for catching that!